website/source/layouts/docs.erb

```diff
@@ -300,6 +300,9 @@
           <li<%= sidebar_current("docs-agent-kv") %>>
             <a href="/docs/agent/kv.html">Consul KV</a>
           </li>
+          <li<%= sidebar_current("docs-agent-sentinel") %>>
+            <a href="/docs/agent/sentinel.html">Sentinel</a>
+          </li>
```

Comment:
The order of this list is a bit of a mystery to me as it is - not alphabetical and I can't quite tell what order it's meant to be, I think originally it was a very rough "order you care about" starting with running and stopping, basic DNS interface, config and then more advanced/optional features later.

I think we kinda confused that with the ACL bits and now it's a bit of a mystery 😄.

If it's order of importance or order in which you might encounter these I'd argue for Sentinel going either right at then end as it's pretty niche currently (and Ent only), or grouped with ACL since that's the only place it currently applies.

All that said I don't feel strongly so this is as good as anywhere until we have a re-organisation of this menu.

My $0.02c is that the Config is the think I navigate to 99% of the time as reference so it used to be easier when it was 2nd in list rather than 4th...

Reply:
Thats really great feedback (I also refer to the config page most often)! I hope to revisit the site navigation again in the near future. I added the Sentinel guide under Consul KV because they seemed like they should be grouped together.